tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -1905,7 +1905,10 @@ public function testUpsertDocument(array $data): void
             'x-appwrite-project' => $this->getProject()['$id'],
         ], $this->getHeaders()), [
             'queries' => [
-                Query::select(['fullName', 'library.*'])->toString(),
+                Query::select([
+                    'fullName',
+                    'library.libraryName',
+                ])->toString(),
```

Comment:
Why do we change this?

Reply:
no specific reason, just wanted to make sure the non-wildcard was working fine in tests as well by only requesting whats needed.

Reply:
Done.